server/games/game.py

```diff
@@ -711,7 +711,14 @@ def get_army_score(self, army):
         return score
 
     def get_army_result(self, player):
-        return self._results.get(self.get_player_option(player.id, 'Army'))
+        results = self._results.get(self.get_player_option(player.id, 'Army'))
+        if not results:
+            return None
+        counts = defaultdict(int)
```

Comment:
Plain dict with setdefault would work just as well. Or I guess if we're using collections anyway, we could just use a Counter for the whole thing.

Reply:
what do you mean by "plain dict with setdefault" ?

Reply:
This is a throwaway dict, so I don't see what kind of data you'd need to write into it "later on". You're using a class from collections anyway (defaultdict), so you might as well use the one that's explicitly made for the thing you want to do. If anything, it's less code.